.github/workflows/.lint_python.yml

```diff
@@ -0,0 +1,26 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
We surely don't need to test across all three platforms.

Reply:
Dropped macOS and Windows